tests/e2e/transaction/helper.ts

```diff
@@ -189,14 +190,14 @@ export const rawTransactionMultiAgentHelper = async (
       transaction: generatedTransaction,
     });
     transactionResponse = await aptos.transaction.submit.multiAgent({
-      transaction: generatedTransaction,
+      transaction: generatedTransaction as MultiAgentTransaction,
```

Comment:
why casting?

Reply:
Because `submit.multiAgent` requires a `MultiAgentTransaction` that includes secondary signers.
If it's not, then you should use `submit.simple`.

In this case `generatedTransaction` can be either type